Formula/at-spi2-core.rb

```diff
@@ -0,0 +1,70 @@
+class AtSpi2Core < Formula
+  desc "Protocol definitions and daemon for D-Bus at-spi"
+  homepage "https://www.freedesktop.org/wiki/Accessibility/AT-SPI2"
+  url "https://download.gnome.org/sources/at-spi2-core/2.36/at-spi2-core-2.36.1.tar.xz"
+  sha256 "97417b909dbbf000e7b21062a13b2f1fd52a336f5a53925bb26d27b65ace6c54"
+  license "LGPL-2.1-or-later"
+  revision 3
+
+  depends_on "gobject-introspection" => :build
+  depends_on "intltool" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python@3.9" => :build
+  depends_on "dbus"
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "libx11"
+  depends_on "libxtst"
+  depends_on :linux
+  depends_on "xorgproto"
+
+  def install
+    ENV.refurbish_args
+
+    mkdir "build" do
+      system "meson", "--prefix=#{prefix}", "--libdir=#{lib}", ".."
+      system "ninja"
+      system "ninja", "install"
+    end
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      /*
+       * List the applications registered on at-spi.
+       */
+
+      #include <atspi/atspi.h>
+      #include <stdlib.h>
+      #include <unistd.h>
+      #include <string.h>
+
+
+      int main(int argc, gchar **argv)
+      {
+        gint i;
+        AtspiAccessible *desktop = NULL;
+        AtspiAccessible *app = NULL;
+
+        atspi_init ();
+
+        desktop = atspi_get_desktop (0);
+        for (i = 0; i < atspi_accessible_get_child_count (desktop, NULL); i++) {
+          app = atspi_accessible_get_child_at_index (desktop, i, NULL);
+
+          g_print ("(Index, application, application_child_count)=(%d,%s,%d)\\n",
+                   i, atspi_accessible_get_name (app, NULL), atspi_accessible_get_child_count (app, NULL));
+          g_object_unref (app);
+        }
+
+        return 1;
+      }
+    EOS
+
+    pkg_config_cflags = shell_output("pkg-config --cflags --libs atspi-2").chomp.split
+    system ENV.cc, "test.c", *pkg_config_cflags, "-lgobject-2.0", "-o", "test"
```

Comment:
```suggestion
    safe_system ENV.cc, "test.c", *pkg_config_cflags, "-lgobject-2.0", "-o", "test"
```
or
```suggestion
    assert_equal true, system ENV.cc, "test.c", *pkg_config_cflags, "-lgobject-2.0", "-o", "test"
```

Reply:
Should we be using `safe_system` in general for system calls moving forward?  We will have to fix many dozens of tests for Linux and could use that as an opportunity to make this change as well.

Reply:
No, `system` in formulae is a totally different thing than `system` used elsewhere so I don't think this needs to change.

`Formula#system` is more less equivalent to `Homebrew.safe_system` + logs.

Reply:
@bo98 is right, ignore me!